help.md

```diff
@@ -0,0 +1,94 @@
+# User Guide
+
+> Welcome! This guide will walk you through the basics of using Zulip Desktop.
+
+
+## Get Zulip Desktop
+
+
+
+## Connect to a Server
+
+### Connect through a proxy
+
+It's possible to connect to your server through a proxy. 
+You can enter the proxy settings in the `Network` section of App Settings. 
+There are three fields provided:
+* `PAC script` - The URL associated with the PAC file.
+* `Proxy rules` - Rules indicating which proxies to use.
+* `Proxy bypass rules` - Rules indicating which URLs should
+    bypass the proxy settings.
+
+Sets the proxy settings.
```

Comment:
I would add a simple example here, which will work for typical setups:

For a typical setup where internet access is required to use an HTTP proxy,
but URLs on the local network should be accessed directly, configure as follows:

`Proxy rules` = `proxy.example.com`
 - Your HTTP proxy server
`Proxy bypass rules` = `*.example.com;10.0.0.0/8`
 - Directly connect to your own domain and private IP subnet

For more complex setups, read below to configure complex proxy rules and proxy bypass rules.